src/main/java/org/jenkinsci/plugins/docker/traceability/core/DockerTraceabilityReportListenerImpl.java

```diff
@@ -72,10 +72,11 @@ private void processEvent(@Nonnull DockerTraceabilityReport report) throws IOExc
                
         // Update containerInfo if available
         final InspectContainerResponse containerInfo = report.getContainer();
-        @CheckForNull Fingerprint containerFP = null;
+        final @CheckForNull Fingerprint containerFP;
```

Comment:
this looks bad as it is unused outside of the if statement below, uninitialized outside of the if statement, and only initialised once within the if statement.
I'm somewhat suprised that it compiles and passed findbugs.